CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+Changelog
+=========
+Version 0.1.0
+
+				Initial release
```

Comment:
I think hard tabs are required in order for the markdown to properly render. When I replace with spaces, it does not look pretty like this:
https://github.com/Stanford-Online/xblock-free-text-response/blob/azimpradhan/add-srm-freetextresponse/CHANGELOG.md